src/core/game/RailNetwork.ts

```
@@ -0,0 +1,327 @@
+      5000,
+      20,
+      new RailNetworkMapAdapter(this.mg),
+    );
+    let fullPath: TrainStation[] = [];
+    switch (this.stationAStar.compute()) {
+      case PathFindResultType.Completed:
+        fullPath = this.stationAStar.reconstructPath();
+        break;
+      case PathFindResultType.Pending:
+        break;
+      case PathFindResultType.PathNotFound:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)